opennms-base-assembly/src/main/filtered/etc/syslogd-configuration.xml

```diff
@@ -14,6 +14,7 @@
             matching-group-message="8"
             discard-uei="DISCARD-MATCHING-MESSAGES"
             timezone=""
+            includeRawSyslogmessage="false"
```

Comment:
Can we omit changing this file, and set a default in the code and XSD instead?

(Avoid changing the configuration files for new optional parameters to help reduce conflicts when upgrading.)